python/tests/custom/test_custom_operations.py

```diff
@@ -107,10 +107,7 @@ def test_toffoli():
 
 
 # NOTE: Ref - https://github.com/NVIDIA/cuda-quantum/issues/1925
-@pytest.mark.parametrize("target", [
-    'density-matrix-cpu', 'nvidia', 'nvidia-fp64', 'nvidia-mqpu',
-    'nvidia-mqpu-fp64', 'qpp-cpu'
-])
+@pytest.mark.parametrize("target", ['density-matrix-cpu', 'nvidia', 'qpp-cpu'])
```

Comment:
To maintain the same coverage, can we create a list of pairs: (target name, [target options])?
e.g., `[('density-matrix-cpu', []), ('nvidia', ['fp32', 'fp64', 'mqpu,fp32', 'mqpu,fp64']), ('qpp-cpu', [])]` 

If the second element of the pair is not empty, then we add it as `option` to the `set_target` call.